extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/JaegerPropagator.java

```diff
@@ -206,7 +204,7 @@ public static JaegerPropagator getInstance() {
   private static SpanContext buildSpanContext(String traceId, String spanId, String flags) {
     try {
       int flagsInt = Integer.parseInt(flags);
-      TraceFlags traceFlags = ((flagsInt & 1) == 1) ? SAMPLED_FLAGS : NOT_SAMPLED_FLAGS;
+      boolean traceFlags = ((flagsInt & 1) == 1) ? SAMPLED_FLAGS : NOT_SAMPLED_FLAGS;
```

Comment:
Nit but we should rename these variables to `sampled` throughout

Reply:
agreed. will do.

Reply:
done. I think I caught a few of them. let me know if there are others I missed